### ISSUE TITLE
Flight controller

### ISSUE THREAD

**Issue by noorbeast** [thestonefox/VRTK#117]
Is it possible to include flight control capability, similar to here: https://www.assetstore.unity3d.com/en/#!/content/61830

**Comment by thestonefox** [comment]
Yeah it was something I was considering adding. I'll leave this issue open as the marker for the task.

**Comment by MrJimbones** [comment]
I'm using this same flying system prefab with the VRTK just fine. After the addition of the ChildOfControllerOnGrab it's actually pretty cherry. ;D

**Comment by stevenblakedesign** [comment]
+1 for flight, just a thought but perhaps helpful to have options to toggle between averaging direction of: 
- controllers only 
- controllers + HMD
- HMD only

**Comment by thestonefox** [comment]
@Fuseman implemented this in one of his tutorials

https://www.youtube.com/watch?v=p9u3yLUR5CQ

Looks relatively easy to implement.

**Comment by kristianpd** [comment]
I think this can be closed, there is no real action to be taken on this right now.

**Comment by reznovVR** [comment]
Seems kinda trivial to add to the new movement actions. However whats the demand for this and more specifically what are the use cases for the people who would like this feature? 

Personally I can see this useful even during development or testing as a god mode but that wouldn't require any collision or other advanced features so the initial implementation would be pretty simple.

Any comments?

**Comment by thestonefox** [comment]
Linked to https://github.com/thestonefox/VRTK/issues/1522

**Comment by thestonefox** [comment]
Actually this is superman flying, i think it's relatively straight forward to implement this sort of feature so we may as well
